### DISCUSSION TITLE
AnimationPlayer copy and paste keyframes

### DISCUSSION THREAD

**Discussion by InTheProcess** [godotengine/godot-proposals#9865] [Editor]
I saw that this issue was resolved previously, but as of Godot 4.2.2, this feature is missing.  Is this a mistake?  I don't want to manually edit the scene file to copy and paste my key frames from one track to another.

**Comment by theraot** [comment]
It is for 4.3, which is in beta at the time of writing. Note that 4.3 has some breaking changes, and although it is in feature freeze now, consider upgrading to it and from it to the final release as risky (make sure you have your project backups/version control).

See https://godotengine.org/article/dev-snapshot-godot-4-3-beta-1/

You can download the beta, and - just as with any other version - you can report any bugs you find at https://github.com/godotengine/godot/issues.

*Plenty of Godot users follow the bleeding edge of dev releases so can use the new features early, and also report bugs.*

**Reply by InTheProcess** [reply]
Thank you!
